v2/tools/generator/internal/jsonast/jsonast.go

```diff
@@ -494,7 +506,9 @@ func getProperties(
 		if property == nil {
 			// TODO: This log shouldn't happen in cases where the type in question is later excluded, see:
 			// TODO: https://github.com/Azure/azure-service-operator/issues/1517
-			klog.V(2).Infof("Property %s omitted due to nil propType (probably due to type filter)", propName)
+			log.V(2).Info(
```

Comment:
Same `v(1)` question

Reply:
and below -- maybe I didn't realize what `--verbose` meant. I thought it was `V(0)` == always and `V(1)` == `--verbose` for these cmdline tools, and that there was no log level "above" that which would be shown. Is that wrong?